src/main/java/com/marklogic/client/modulesloader/impl/BaseModulesFinder.java

```diff
@@ -31,6 +31,8 @@
     private String optionsPath = "options";
     private String namespacesPath = "namespaces";
     private String transformsPath = "transforms";
+    // special case for excluding from assets.
+    private String schemasPath = "schemas";
```

Comment:
I saw in the ml-app-deployer PR that the schemas folder was under src/main/ml-config, but this change implies that the folder would be under src/main/ml-modules. I think I like the latter more - schemas are modules and they seem more like code vs configuration, so src/main/ml-modules/schemas seems like the most intuitive place (I was initially thinking they'd be under ./data/schemas, but that doesn't seem right anymore - I think a "./data" folder is best reserved for stuff that goes into your content database). 

They could also go under "src/main/schemas" or "src/main/ml-schemas" - this seems somewhat consistent with Roxy, which I believe stores them under "./schemas" (and I think the top-level folder in a Roxy project is roughly similar to "src/main" in a Java project). 

"src/main/ml-schemas" might be best - perhaps the one downside to src/main/ml-modules/schemas is that we now have a single special folder that doesn't get loaded into the modules database, but rather into the schemas database. If we don't do that, then we don't need the change in this class. 

Sorry to be unsure about this, I just have never used schemas and thus don't know how best to organize/separate them from modules, config, and data.